### PR TITLE
[Closes #2] Add method that requires the variable not to be nil

### LINKED ISSUE #2: Add get! method
Add a `get!` method that raises an exception if the variable being fetched is nil.

### PR DESCRIPTION
**Pull request by lhoppe**

### CODE REVIEW

File: spec/konfa_spec.rb [hunk 1]
```diff
@@ -91,6 +91,18 @@ def allowed_variables
     end
   end
 
+  context '#get!' do
+    it 'can access a variable that is not nil' do
```
Comment (ghansson) [inline spec/konfa_spec.rb]: Just to cover all the bases, should we add tests checking outcome for things like empty string and the word "nil" and things? To provide face-palm protection for future fixes and refactorings

Comment (lhoppe) [inline spec/konfa_spec.rb]: Could be good to have I suppose. I'll add those.
